src/Workspaces/MSBuildTest/Workspaces.MSBuild.Test.csproj

```diff
@@ -17,12 +17,18 @@
     <ProjectReference Include="..\..\Compilers\CSharp\Portable\CSharpCodeAnalysis.csproj" />
     <ProjectReference Include="..\..\Compilers\Test\Resources\Core\CompilerTestResources.csproj" />
     <ProjectReference Include="..\..\Compilers\VisualBasic\Portable\BasicCodeAnalysis.vbproj" />
+    <ProjectReference Include="..\..\EditorFeatures\Core\EditorFeatures.csproj" />
```

Comment:
Any idea which project reference was added that required this to be added? BuildBoss is forcing transitivity which I get, but still seems odd our MSBuild workspace tests depends on IDE code, even through some indirect chain?

Reply:
It's because WorkspaceTestServices now references EditorFeatures. I'm not a fan of this reference. It feels like a layering violation. This is from @sharwell's recent changes for unit tests using MEF.